### ISSUE TITLE
Can't connect to DB from symfony app

### ISSUE THREAD

**Issue by ppi-buck** [dockerwest/compose-symfony#7]
Hey again,

testing this tool as well ;-) 
Everything here works as intended up to the point where I try to connect my CMS (Contao 4.5) with my database. In the docker-compose it looks to me that you should use `mysql.application.docker` as the host `root` as the user and `toor` as the default password. (As I'm only trying out things here locally I didn't change those yet).

I'm getting this error:
![image](https://user-images.githubusercontent.com/37330764/40824117-363c576e-6573-11e8-9964-ff1944347b8b.png)

I also tried other hosts that seemed reasonable to me like `mysql`, `localhost`, `127.0.0.1` etc. None did the trick. 

What am I missing here? Or is Contao not compatible with your tool? It should in my understanding as it's a symfony 3 application. Btw: I did change the symfony version in .env to match.

Thanks for your help,
Cheers

**Comment by ppi-buck** [comment]
Also installing via or accessing the contao-manager in a pre-installed version isn't possible. It returns a 404.
Maybe it is connected in a way?! I think it's inside the nginx settings that I can't access the manager-file. How can I tell nginx to allow this file? I tried the adding the following to `/etc/nginx/include/overrides.conf` right in my environment directory. 

```
server {
    
    location ~ ^/(test)\.php(/|$) {
        # set upstream
        set $upstream application:9000;
        fastcgi_pass   $upstream;

        fastcgi_index  index.php;
        fastcgi_split_path_info ^(.+\.php)(/.+)$;
        fastcgi_param  SCRIPT_FILENAME    $document_root$fastcgi_script_name;
        fastcgi_param DOCUMENT_ROOT $realpath_root;
        include        fastcgi_params;
        # Set a maximum execution time. This should be the same as your php_max_exec time and php-fpm pool request_terminate_timeout to prevent
        # any gateway timeouts and/or worker processes being tied up.
        fastcgi_read_timeout 60;
    }

    location ~ ^/(contao-manager\.phar)\.php(/|$) {
        # set upstream
        set $upstream application:9000;
        fastcgi_pass   $upstream;

        fastcgi_index  index.php;
        fastcgi_split_path_info ^(.+\.php)(/.+)$;
        fastcgi_param  SCRIPT_FILENAME    $document_root$fastcgi_script_name;
        fastcgi_param DOCUMENT_ROOT $realpath_root;
        include        fastcgi_params;
        # Set a maximum execution time. This should be the same as your php_max_exec time and php-fpm pool request_terminate_timeout to prevent
        # any gateway timeouts and/or worker processes being tied up.
        fastcgi_read_timeout 60;
    }
}
```
Seems to change nothing, can't open the test.php nor the contao-manager.phar.php - even if I try to break the `.conf` the nginx service starts up just fine. Guessing wrong file location?!

Thanks for helping.
Cheers

**Comment by MVermue** [comment]
I also set up this docker for one of my projects yesterday, and this is what I had to do to MySQL:

In your `docker-compose.yml` and `docker-compose-dinghy.yml` file change `image: mysql` to `image: mysql:5.7`. Since the version wasn't specified before, it tried to take the latest version (which is now MySQL 8) which I don't think Doctrine fully supports yet.
The MySQL host parameter should be mysql.<whatever you set as BASEHOST in the .env file of your docker> (if your BASEHOST is set to `application.docker`, MySQL would be on `mysql.application.docker`). 

After having run the `environment` script in this repository you have some extra scripts available to use, like `create_db contao` which would create the database for you so you can connect to it (might have to wait 20 seconds or so after docker says the container started for it to properly work though).

**Comment by ppi-buck** [comment]
Thanks! You are genious! 
Didn't even think about the MySQL version. Totally works now. :-) 

If this is a general issue with doctrine and MySQL maybe it's a good idea to restrict the version in the default docker-compose.yml already so this repo can be used as is. Up for the devs @dockerwest to decide.

**Comment by BlackIkeEagle** [comment]
Ok hearing this we will have to restrict all mysql to 5.7

**Comment by BlackIkeEagle** [comment]
this is now fixed, we set mysql fixed at mysql 5.7 for now
